python/mxnet/visualization.py

```diff
@@ -134,20 +134,26 @@ def print_layer_summary(node, out_shape):
                             pre_filter = pre_filter + int(shape[0])
         cur_param = 0
         if op == 'Convolution':
-            if ("no_bias" in node["attrs"]) and int(node["attrs"]["no_bias"]):
-                cur_param = pre_filter * int(node["attrs"]["num_filter"])
+            if "no_bias" in node["attrs"] and node["attrs"]["no_bias"] == 'True':
+                num_group = int(node["attrs"]["num_group"]) if \
+                   "num_group" in node["attrs"] else 1
```

Comment:
int(node['attrs'].get('num_group', '1')). same below.

Reply:
`dict.get` handles default value for missing value so the trailing `if` is no longer necessary.

Reply:
Ok